robot_descriptions/loaders/idyntree.py

```diff
@@ -39,6 +47,18 @@ def load_robot_description(description_name: str) -> int:
         raise ValueError(f"{description_name} is not a URDF description")
 
     model_loader = idyn.ModelLoader()
-    model_loader.loadModelFromFile(module.URDF_PATH)
+
+    if (joints_list is None):
```

Comment:
Hmm not sure this is Black...

Oh well, actually Black is not configured properly in the CI :see_no_evil: (It's missing ``--check``.)

Never mind formatting for this PR, I'll enable it and re-format subsequently.